### ISSUE TITLE
stacktrace from CheckCollection internals

### ISSUE THREAD

**Issue by wireframe** [sportngin/okcomputer#129]
version: 1.14.1

configuration:
```
# config/initializers/okcomputer.rb
OkComputer::Registry.register :sidekiq_latency_check, OkComputer::SidekiqLatencyCheck.new(:default, 60)
```

stacktrace:
```
ArgumentError: comparison of OkComputer::ActiveRecordCheck with OkComputer::SidekiqLatencyCheck failed
  from ok_computer/check_collection.rb:85:in `sort'
  from ok_computer/check_collection.rb:85:in `to_text'
  from action_controller/metal/rendering.rb:116:in `block in _normalize_text'
  from action_controller/metal/rendering.rb:114:in `each'
  from action_controller/metal/rendering.rb:114:in `_normalize_text'
  from action_controller/metal/rendering.rb:84:in `_normalize_options'
  from abstract_controller/rendering.rb:128:in `_normalize_render'
  from abstract_controller/rendering.rb:25:in `render'
  from action_controller/metal/rendering.rb:36:in `render'
  from action_controller/metal/instrumentation.rb:44:in `block (2 levels) in render'
  from active_support/core_ext/benchmark.rb:12:in `block in ms'
  from benchmark.rb:308:in `realtime'
  from active_support/core_ext/benchmark.rb:12:in `ms'
```

**Comment by pbyrne** [comment]
I suspect the problem is in registering the check with a symbol for its name. When sorting the checks by name, it runs into trouble. If you switch to `"sidekiq_latency_check"` instead of `:sidekiq_latency_check` when registering, does this problem go away?

Assuming that's the problem, we should do a better job and coerce the names into strings, regardless of how they're registered (or compare the string value when sorting checks).

**Comment by pbyrne** [comment]
A fix for this error has been shipped as v1.4.2.
